.gitignore

```diff
@@ -13,6 +13,7 @@ dist/
 venv/
 htmlcov/
 .coverage*
+coverage.xml
```

Comment:
Why do we have to add this?  Gunk left over by the `codecov` command's invoking `coverage xml`?